packages/metrics-server/package.yaml

```diff
@@ -0,0 +1,5 @@
+url: https://github.com/helm/charts.git
```

Comment:
@galal-hussein Can we switch to use tgz files links? doing git clone is quite expensive.

Reply:
sure